src/lib/health-checks.js

```diff
@@ -5,11 +5,14 @@
  * @typedef {import("../../typings/n-express").AppOptions} AppOptions
  */
 
+const nLogger = require('@financial-times/n-logger').default;
+
 const errorRateCheck = require('./error-rate-check');
-const unRegisteredServicesHealthCheck = require('./unregistered-services-healthCheck');
-const metricsHealthCheck = require('./metrics-healthcheck');
 const herokuLogDrainCheck = require('./heroku-log-drain-check');
-const nLogger = require('@financial-times/n-logger').default;
+const metricsHealthCheck = require('./metrics-healthcheck');
+const supportedNodeJsVersionCheck = require('./supported-node-js-version-check');
+const unRegisteredServicesHealthCheck = require('./unregistered-services-healthCheck');
```

Comment:
The net change here is simply the addition of `const supportedNodeJsVersionCheck = require('./supported-node-js-version-check');`; otherwise it is separating external dependencies from native ones, and alphabetising the native dependencies.